rest-service/manager_rest/deployment_update/handlers.py

```diff
@@ -184,6 +187,51 @@ def modify(self, ctx, current_entities):
                 self._modify_relationship_operation,
                 self._modify_node_operation)
 
+    def _update_stored_operations(self, ctx, node, new_operation):
+        """Update the operations table with the new operation inputs."""
+
+        def _filter_operation(column):
+            # path in the parameters dict that stores the node name
+            node_name_path = ('task_kwargs', 'cloudify_context', 'node_name')
+            # this will use postgres' json operators
+            return cast(column, JSON)[node_name_path].astext == node.id
+
+        if not ctx.deployment_update.update_executions:
+            return
+        executions = self.sm.list(models.Execution, filters={
+            'deployment_id': ctx.deployment_update.deployment_id,
+            'status': [
+                ExecutionState.PENDING,
+                ExecutionState.STARTED,
+                ExecutionState.CANCELLED,
+                ExecutionState.FAILED
+            ]
+        })
+        graphs = self.sm.list(models.TasksGraph, filters={
+            'execution_id': [e.id for e in executions]
+        })
+        # update only those operations - for the current node, ones that
+        # in a state which can be resumed, and for the current operation
+        ops = self.sm.list(models.Operation, filters={
+            'parameters': _filter_operation,
+            'name': new_operation['operation'],
+            '_tasks_graph_fk': [tg._storage_id for tg in graphs],
+            'state': [cloudify_tasks.TASK_RESCHEDULED,
+                      cloudify_tasks.TASK_FAILED,
+                      cloudify_tasks.TASK_PENDING]
+        })
+        new_op_inputs = new_operation['inputs']
+        for op in ops:
+            try:
+                op.parameters['task_kwargs']['kwargs'].update(new_op_inputs)
+                op.parameters['task_kwargs']['kwargs']['__cloudify_context'][
+                    'has_intrinsic_functions'] = True
+                op.parameters['task_kwargs']['cloudify_context'][
+                    'has_intrinsic_functions'] = True
```

Comment:
it's currently in these two places, CY-1711 is about removing one of those places

Reply:
this can be removed after cloudify-cosmo/cloudify-common#361